feature-libs/cart/main/root/components/mini-cart/mini-cart-component.service.ts

```diff
@@ -0,0 +1,123 @@
+import { Injectable } from '@angular/core';
+import {
+  AuthService,
+  BASE_SITE_CONTEXT_ID,
+  EventService,
+  FacadeFactoryService,
+  SiteContextParamsService,
+  StatePersistenceService,
+  UnifiedInjector,
+} from '@spartacus/core';
+import { combineLatest, Observable, of } from 'rxjs';
+import {
+  distinctUntilChanged,
+  filter,
+  map,
+  startWith,
+  switchMap,
+  takeWhile,
+} from 'rxjs/operators';
+import { CartConfig } from '../../config/cart-config';
+import { ActiveCartFacade } from '../../facade/active-cart.facade';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MiniCartComponentService {
+  constructor(
+    protected activeCartFacade: ActiveCartFacade,
```

Comment:
please clean the constructor parameters, some of them are not used.

Reply:
Thanks.  Good catch.

Reply:
Fixed